### PR TITLE
change StopTimeout type

### PR DESCRIPTION
**Pull request by carolynhu**

### CODE REVIEW

File: convert/convert.go [hunk 1]
```diff
@@ -166,7 +166,7 @@ func Convert(c *config.ServiceConfig, ctx project.Context) (*container.Config, *
 		Volumes:      toMap(Filter(vols, isVolume)),
 		MacAddress:   c.MacAddress,
 		StopSignal:   c.StopSignal,
-		StopTimeout:  &[]int64{int64(c.StopGracePeriod)}[0],
+		StopTimeout:  &[]int{int(c.StopGracePeriod)}[0],
```
Comment (joshwget) [inline convert/convert.go:169]: Does `&c.StopGracePeriod` work here?

**Comment by joshwget** [comment]
LGTM

**Review by joshwget — APPROVED** [review]

**Comment by StrongMonkey** [comment]
~~should we change the vendor here? i think the root reason in here is that the docker library in executor differs from what is in cli.~~

**Comment by carolynhu** [comment]
@StrongMonkey no, we'd better not change vendor, so I change it back, *int is the type for StopTimeout in docker library

**Pull request merged**